tools/backup/templates/backup.yaml

```diff
@@ -57,17 +60,21 @@ spec:
                   value: "{{ .Values.checkLabelScanStore }}"
                 - name: CHECK_PROPERTY_OWNERS
                   value: "{{ .Values.checkPropertyOwners }}"
+              {{- if .Values.secretName }}
```

Comment:
`activate_aws` function in `backup.sh` expect credentials file if a user does not supply `secretName` then it will throw an error message for credentials file.

Reply:
yes, that's correct. The was implementation requires `secretName`. In this PR I have only added GCP support for workload identity (meaning service account based authentication rather than using credentials in a secret).

Reply:
AWS implementation will also not require `secretName` if you have SA with proper role and permission. We have tried it with alpine image and if you mount SA and SA token then you don't need `secretName`.

Reply:
Ok I can make the necessary change

Reply:
we just noticed that even if the pod uses a service account it needs a region to communicate with the AWS service. We will need the credential file in the case of AWS to get the region.

Reply:
OK - in that case I'd like a bit more opportunity to investigate what we can do in AWS, rather than trying to make further changes in this PR.

If you want to discuss this further could you create a new issue for "improvements to accessing to AWS S3 for backup & restore"?

Reply:
For example I think you can access S3 without needing any credentials in the Pod if you can make use of a VPC  Endpoint gateway for S3:
https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html

Reply:
You can ignore this change, it is not a priority right now.